fluentd/lib/fluent-plugin-viaq_data_model/lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -181,6 +196,19 @@ def configure(conf)
         raise Fluent::ConfigError, "Field [#{@src_time_name}] must be listed in default_keep_fields or extra_keep_fields"
       end
       @undefined_dot_replace_char = nil if @undefined_dot_replace_char == DOT_REPLACE_CHAR_UNUSED
+      
+      if @levels
+        buffer = []
+        @levels.each_with_index do |level,i|
+          if i < 10 
+            buffer.append("?<l#{i}_#{level.name}>#{level.match}")
+          else
+            log.info("Levels supports a maximum of 10 matches. Ignoring: #{level}")
```

Comment:
Why 10? Why a fixed limit at all?

Reply:
only to make it easier to do the substring of the prefix.  It was a convenient way to not need to figure out the logic